lms/djangoapps/grades/api/v0/tests/test_views.py

```diff
@@ -0,0 +1,203 @@
+"""
+Tests for the views
+"""
+from datetime import datetime
+
+from django.core.urlresolvers import reverse
+from mock import patch
+from pytz import UTC
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase, TEST_DATA_SPLIT_MODULESTORE
+
+
+class CurrentGradeViewTest(SharedModuleStoreTestCase, APITestCase):
```

Comment:
> invalid course-id (`InvalidKeyError` raised) 

Theoretically this should never happen because the regex in the URL should take care of it. I will just mock CourseKey to raise.
I will add also a test for a valid key but inexistent course